proto/meta.proto

```diff
@@ -262,9 +262,9 @@ message SubscribeResponse {
   uint64 version = 3;
   oneof info {
     common.WorkerNode node = 4;
-    catalog.Database database_v2 = 8;
-    catalog.Schema schema_v2 = 9;
-    catalog.Table table_v2 = 10;
+    catalog.Database database = 8;
```

Comment:
Maybe also fix the sequence?

Reply:
So we don't need to do forward compatibility yet? 🤣

Reply:
No, not yet I guess. 🤣